src/Resources/contao/templates/survey/survey_answers_matrix.html5

```diff
@@ -10,11 +10,25 @@
 </thead>
 <tbody>
 <?php $colcounter = 1; ?>
-<?php foreach ($this->choices as $choiceid => $choice): ?>
+<?php foreach ($this->choices as $choiceid => $choice):
+dump(array_key_exists($counter, $this->statistics['cumulated']));
```

Comment:
You forgot removing a test output :)

Reply:
yes - sorry - please remove this stuff during merge ☺️